app/geo/gui/src/ts/twxplore/gui/geo/featureTypeStrategies/PolygonFeatureTypeStrategy/PolygonFeatureTypeStrategy.ts

```diff
@@ -1,9 +1,41 @@
 import {FeatureTypeStrategy} from "../FeatureTypeStrategy";
 import {FeatureType, FeatureQuery} from "../../api/graphqlGlobalTypes";
 import {FeatureAttributeName} from "../../states/map/FeatureAttributeName";
+import {Dispatch} from "redux";
+import {FeaturesByType} from "../../states/map/FeaturesByType";
+import {MapFeatureTypeState} from "../../states/map/MapFeatureTypeState";
+import {layerConfigChange, interactionConfigChange} from "kepler.gl/actions";
 
 export abstract class PolygonFeatureTypeStrategy
   implements FeatureTypeStrategy {
+  dispatchLayerConfigurationActions(
+    keplerLayerOfFeatureType: any,
+    keplerFiltersOfFeatureType: any,
+    keplerFieldsOfFeatureType: any,
+    keplerInteractionConfigOfFeatureType: any,
+    dispatch: Dispatch<any>,
+    featuresByType: {
+      [featureType: string]: FeaturesByType;
+    }
+  ): void {
+    switch (featuresByType[this.name].featureTypeState) {
```

Comment:
The implementations all appear to be doing this expression, just pass it in instead of featuresByType?